frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -382,6 +422,13 @@ export const transformTopologyData = (
     graph: { nodes: [], edges: [], groups: [] },
     topology: {},
   };
+  if (trafficData)
```

Comment:
You modified the output here and it didn't fail a test 😕 I think we need to write a test for this condition too.. otherwise we'll not catch a break if this violates the status quo.

Reply:
Even if this condition is not true it doesn't affect the edges array. it's just that Traffic connector will be not be added to the edges array.

Reply:
Ah, doh - right it wouldn't fail pre-existing tests. My brain is half functioning today 😆 But we still need to write a test for if it is truthy.

Reply:
yes I did for falsy

Reply:
I'm confused by this comment:

> yes I did for falsy

This didn't exist during my last test, did it? I'm 99% I didn't see the falsy test lol.

Reply:
No, there wasn't any test to check the condition. But now I added for both falsy and truthy. :wink: